examples/linear_model/plot_ols_ridge_variance.py

```diff
@@ -47,7 +47,7 @@
     fig = plt.figure(fignum, figsize=(4, 3))
     plt.clf()
     plt.title(name)
-    ax = plt.axes([.12, .12, .8, .8])
+    ax = plt.axes([.10, .10, .8, .8])
```

Comment:
I feel using `ax` here makes the plot more complicated. I've played around with this example and it seems that not setting `plt.axes` and instead just using `plt` works well.

Reply:
Yes, I think we can get what we want with something very simple (e.g., plt.subplot -> plt.plot -> plt.title -> plt.axis)

Reply:
Yes, that piece can be simplified. Thanks for your inputs. @qinhanmin2014 @xhlulu